tests/app/questionnaire/test_navigator.py

```diff
@@ -48,19 +59,39 @@ def test_introduction_not_in_path_when_not_in_schema(self):
 
         navigator = Navigator(survey)
 
-        blocks = [b['block_id'] for b in navigator.get_location_path()]
+        blocks = [b.block_id for b in navigator.get_location_path()]
 
         self.assertNotIn('introduction', blocks)
 
     def test_next_with_conditional_path(self):
         survey = load_schema_file("0_star_wars.json")
 
         expected_path = [
-            {"block_id": "f22b1ba4-d15f-48b8-a1f3-db62b6f34cc0"},
-            {"block_id": "96682325-47ab-41e4-a56e-8315a19ffe2a"},
-            {"block_id": "cd3b74d1-b687-4051-9634-a8f9ce10a27d"},
-            {"block_id": "an3b74d1-b687-4051-9634-a8f9ce10ard"},
-            {"block_id": "846f8514-fed2-4bd7-8fb2-4b5fcb1622b1"}
+            Location(
```

Comment:
Guh, could've asked for these in my other PR, there's 301 of them! See you tomorrow.